sdk/src/main/scala/com/horizen/account/block/AccountBlock.scala

```diff
@@ -94,7 +93,7 @@ class AccountBlock(override val header: AccountBlockHeader,
 }
 
 
-object AccountBlock extends SparkzEncoding {
+object AccountBlock {
```

Comment:
can you check the usage of `verifyLogsBloomConsistency`?
it seems it was incidentally removed from the AccountState during one of the merges/prs

Reply:
Ok, restored